protractor.conf.ts

```diff
@@ -206,15 +207,10 @@ exports.config = {
     baseUrl: HOST,
 
     params: {
-        config: appConfig,
-        identityAdmin: {
-            email: IDENTITY_ADMIN_EMAIL,
-            password: IDENTITY_ADMIN_PASSWORD
-        },
-        identityUser: {
-            email: USERNAME_ADF,
-            password: PASSWORD_ADF
-        }
+        testConfig: TestConfig,
+        config: TestConfig.appConfig,
+        identityAdmin: TestConfig.identityAdmin,
+        identityUser: TestConfig.identityUser,
```

Comment:
I see this part a little confusing, some users are coming from testConfig and other ones (cloud users/admins) can be accessed directly. Do we really need these identity people there?